metadata_service/proxy/atlas_proxy.py

```diff
@@ -72,37 +74,61 @@ def _extract_info_from_uri(self, *, table_uri: str) -> Dict:
         result = pattern.match(table_uri)
         return result.groupdict() if result else dict()
 
-    def get_user_detail(self, *, user_id: str) -> Union[UserEntity, None]:
-        pass
-
-    def get_table(self, *, table_uri: str) -> Table:
+    def _get_table_entity(self, *, table_uri: str) -> Tuple:
         """
-        Gathers all the information needed for the Table Detail Page.
-        It tries to get the table information from
+        Fetch information from table_uri and then find the appropriate entity
+        The reason, we're not returning the entity_unique_attribute().entity
+        directly is because the entity_unique_attribute() return entity Object
+        that can be used for update purposes,
+        while entity_unique_attribute().entity only returns the dictionary
         :param table_uri:
         :return:
         """
         table_info = self._extract_info_from_uri(table_uri=table_uri)
 
         try:
-            # The reason to use the DATASET_ENTITY here instead of TABLE_ENTITY
-            # is to access the older data (if any) which was generated before
-            # introducing the TABLE_ENTITY in atlas.
-            entity = self._driver.entity_unique_attribute(
+            return self._driver.entity_unique_attribute(
                 table_info['entity'],
-                qualifiedName=table_info.get('name')).entity
+                qualifiedName=table_info.get('name')), table_info
         except Exception as ex:
             LOGGER.exception('Table not found. {}'.format(str(ex)))
             raise NotFoundException('Table URI( {table_uri} ) does not exist'
                                     .format(table_uri=table_uri))
 
+    def get_user_detail(self, *, user_id: str) -> Union[UserEntity, None]:
+        pass
+
+    def get_table(self, *, table_uri: str) -> Table:
+        """
+        Gathers all the information needed for the Table Detail Page.
+        :param table_uri:
+        :return:
+        """
+        entity, table_info = self._get_table_entity(table_uri=table_uri)
+        table_details = entity.entity
+
         try:
+            attrs = table_details['attributes']
+            rel_attrs = table_details['relationshipAttributes']
+
+            tags = []
+            for classification in table_details.get("classifications", list()):
+                tags.append(
+                    Tag(
+                        tag_name=classification.get('typeName'),
+                        tag_type=classification.get('typeName')
```

Comment:
Could you use "default" in tag_type? We are currently not using `tag_type`, but filling it with "default".

Reply:
ah, good to know. will update this, thanks 👍